winpr/libwinpr/thread/process.c

```diff
@@ -213,6 +232,8 @@ BOOL _CreateProcessExA(HANDLE hToken, DWORD dwLogonFlags,
 		lpszEnvironmentBlock = GetEnvironmentStrings();
 		envp = EnvironmentBlockToEnvpA(lpszEnvironmentBlock);
 	}
+	if (!envp)
```

Comment:
lpszEnvironmentBlock is missing a null check after line 232 above. Not sure if that should be part of this pr